### PR TITLE
feat(storage.cold-archive): edit container creation success message

### PR DESCRIPTION
**Pull request by brofik**
ref: MANAGER-11029

| Question         | Answer
| ---------------- | ---
| Branch?          | `master` 
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | Fix # MANAGER-11029 
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] Breaking change is mentioned in relevant commits

### CODE REVIEW

File: packages/manager/modules/pci/src/projects/project/storages/cold-archive/add/translations/Messages_fr_FR.json [hunk 1]
```diff
@@ -11,8 +11,8 @@
   "pci_projects_project_storages_cold_archive_add_action_create": "Créer un conteneur",
   "pci_projects_project_storages_cold_archive_add_action_cancel": "Annuler",
   "pci_projects_project_storages_cold_archive_add_action_create_pending": "Conteneur en cours de création",
-  "pci_projects_project_storages_cold_archive_add_action_create_archive_create_request_success_linked_user": "Le conteneur {{container}} a été créé avec succès. L'utilisateur S3 {{ userName }} est administrateur de ce conteneur.",
+  "pci_projects_project_storages_cold_archive_add_action_create_archive_create_request_success_linked_user": "Le conteneur {{container}} a été créé avec succès et l'utilisateur S3 {{ userName }} a été ajouté en tant que Administrator.",
```
Comment (mohammed-zahaf) [inline packages/manager/modules/pci/src/projects/project/storages/cold-archive/add/translations/Messages_fr_FR.json:14]:
```suggestion
  "pci_projects_project_storages_cold_archive_add_action_create_archive_create_request_success_linked_user": "Le conteneur {{container}} a été créé avec succès et l'utilisateur S3 {{ userName }} a été ajouté en tant qu'Administrator.",
```

File: packages/manager/modules/pci/src/projects/project/storages/cold-archive/add/translations/Messages_fr_FR.json [hunk 2]
```diff
@@ -11,8 +11,8 @@
...
   "pci_projects_project_storages_cold_archive_add_action_create_archive_create_request_success_no_linked_user": "Le conteneur {{container}} a été créé.",
-  "pci_projects_project_storages_cold_archive_add_action_create_archive_create_request_success": "Le conteneur {{containerName}} a été créé avec succès. L'utilisateur S3 {{ userName }} est administrateur de ce conteneur.",
+  "pci_projects_project_storages_cold_archive_add_action_create_archive_create_request_success": "Le conteneur {{containerName}} a été créé avec succès et l'utilisateur S3 {{ userName }} a été ajouté en tant que Administrator.",
```
Comment (mohammed-zahaf) [inline packages/manager/modules/pci/src/projects/project/storages/cold-archive/add/translations/Messages_fr_FR.json:16]:
```suggestion
  "pci_projects_project_storages_cold_archive_add_action_create_archive_create_request_success": "Le conteneur {{containerName}} a été créé avec succès et l'utilisateur S3 {{ userName }} a été ajouté en tant qu'Administrateur.",
```

Comment (brofik) [reply packages/manager/modules/pci/src/projects/project/storages/cold-archive/add/translations/Messages_fr_FR.json:16]: j'ai utilisé la nomenclature utilisée dans la spécification du ticket.

Comment (mohammed-zahaf) [reply packages/manager/modules/pci/src/projects/project/storages/cold-archive/add/translations/Messages_fr_FR.json:16]: i guess we could challenge the PU.
if they are not agree let me know.

**Review by mohammed-zahaf — DISMISSED** [review]

File: packages/manager/modules/pci/src/projects/project/storages/cold-archive/add/translations/Messages_fr_FR.json (hunk 1 above)
Comment (brofik) [reply packages/manager/modules/pci/src/projects/project/storages/cold-archive/add/translations/Messages_fr_FR.json]: This change is outdated.

**Pull request merged**